src/datamapper/AusenciaJpaController.java

```diff
@@ -170,9 +173,9 @@ public Ausencia findAusencia(Long id) {
     public Ausencia findAusencia(String codigo){
     List<Ausencia> ausencias = this.findAusenciaEntities();
     for(Ausencia ausencia : ausencias){
-        if(ausencia.getCodigo().equals(codigo))
+        if(ausencia.getCodigo().equals(codigo)){

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['if' have incorrect indentation level 8, expected level should be 12.](https://www.codacy.com/app/calebepb/substituicao/file/2779925028/issues/source?bid=3364462&fileBranchId=3364487#l176)